src/exports.js

```diff
@@ -188,5 +189,7 @@ export default function makeExports(
         flushToStyleTag,
         injectAndGetClassName,
         defaultSelectorHandlers,
+        reset,
+        resetInjected,
```

Comment:
Can you add documentation for these methods to the readme?

Reply:
Added